hubspot/string.go

```diff
@@ -4,5 +4,5 @@ import "fmt"
 
 // String returns a string representation of the connector, which is useful for logging / debugging.
 func (c *Connector) String() string {
-	return fmt.Sprintf("hubspot.Connector[%s]", c.Module)
```

Comment:
I just noticed this `String` method is not a part of any the interfaces yet, it should probably be a part of BasicConnector

Reply:
`BasicConnector` already has `fmt.Stringer`